### PR TITLE
quickstart: remove optionals settings for certificates, and cookie secrets

### PR DESCRIPTION
**Pull request by ZPain8464**
## Changes

These changes remove the optional settings for certificates, mounting wild-card certs with docker, secrets, and provides workarounds for when the user is presented with a warning for self-signed certificates. 

## Related issues
Closes #141 (as dupe)
Closes https://github.com/pomerium/pomerium/issues/3737

### CODE REVIEW

**Review by desimone — CHANGES_REQUESTED** [review]

File: content/docs/install/quickstart.mdx [hunk 1]
```diff
@@ -42,18 +40,31 @@ docker-compose up
...
+
+To resolve this error:
+1. Click anywhere inside the browser window
+2. Type **thisisunsafe**
+3. Make sure **Reload** is selected
+3. Press **ENTER**
+
+Your browser will redirect you to the route defined in your policy: 
+
+![user details page](./img/user-details-page.png)
```
[the remark this answers is not in the document]
Comment (ZPain8464) [reply content/docs/install/quickstart.mdx:58]: I installed Shottr, WAY cleaner. I'm updating the image.

**Comment by ZPain8464** [comment]
@desimone and @calebdoxsey apologies on the confusion with the HedgeDoc guide and the PoC/Quickstart update – I'll create feature branches for updates going forward. 

I've added cleaner images to both docs and created a HedgeDoc subdir for images. Please let me know if anything else needs updating and I'll be happy to jump on it!

**Comment by calebdoxsey** [comment]
Quickstart LGTM

**Review by desimone — APPROVED** [review]

File: content/docs/install/quickstart.mdx [hunk 2]
```diff
@@ -42,18 +40,31 @@ docker-compose up
...
 
-You should now be able access to the routes (e.g. `https://verify.localhost.pomerium.io`) as specified in your policy file.
+You should now be able to access the routes (e.g. `https://verify.localhost.pomerium.io`) as specified in your policy file.
 
-You can also navigate to the special pomerium endpoint `verify.localhost.pomerium.io/.pomerium/` to see your current user details.
+### Handle HSTS Errors
+When navigating to the `https://verify.localhost.pomerium.io` route defined in your policy, you may encouter the following HSTS error:
 
-![currently logged in user](./img/logged-in-as.png)
+![hsts error](./img/hsts-error.png)
```
Comment (desimone) [inline content/docs/install/quickstart.mdx:48]:
```suggestion
![self-signed warning](./img/hsts-error.png)
```

File: content/docs/install/quickstart.mdx [hunk 3]
```diff
@@ -42,18 +40,31 @@ docker-compose up
 
 Docker will automatically download the required [container images] for Pomerium and [verify]. Then, Pomerium will run with the configuration details set in the previous steps.
 
-You should now be able access to the routes (e.g. `https://verify.localhost.pomerium.io`) as specified in your policy file.
+You should now be able to access the routes (e.g. `https://verify.localhost.pomerium.io`) as specified in your policy file.
 
-You can also navigate to the special pomerium endpoint `verify.localhost.pomerium.io/.pomerium/` to see your current user details.
+### Handle HSTS Errors
+When navigating to the `https://verify.localhost.pomerium.io` route defined in your policy, you may encouter the following HSTS error:
```
Comment (desimone) [inline content/docs/install/quickstart.mdx:46]:
```suggestion
When navigating to the `https://verify.localhost.pomerium.io` route defined in your policy, you may encounter the following self-signed certificate warning:
```

File: content/docs/install/quickstart.mdx [hunk 4]
```diff
@@ -42,18 +40,31 @@ docker-compose up
 
 Docker will automatically download the required [container images] for Pomerium and [verify]. Then, Pomerium will run with the configuration details set in the previous steps.
 
-You should now be able access to the routes (e.g. `https://verify.localhost.pomerium.io`) as specified in your policy file.
+You should now be able to access the routes (e.g. `https://verify.localhost.pomerium.io`) as specified in your policy file.
 
-You can also navigate to the special pomerium endpoint `verify.localhost.pomerium.io/.pomerium/` to see your current user details.
+### Handle HSTS Errors
```
Comment (desimone) [inline content/docs/install/quickstart.mdx:45]:
```suggestion
### Handle Self-Signed Certificate Warning
```

**Review by desimone — CHANGES_REQUESTED** [review]

**Review by desimone — CHANGES_REQUESTED** [review]
Lots of unrelated changes. Looks like you branched off an old main, then did a soft reset?

**Comment by ZPain8464** [comment]
@desimone hopefully this is the final push needed to get this up to date. Requesting review.

**Review by desimone — APPROVED** [review]
🥳

**Pull request merged**